config/routes.rb

```diff
@@ -1,4 +1,11 @@
 Rails.application.routes.draw do
+  root 'users#index'
+  get 'users', to: 'users#index'
+  get 'users/:id', to: 'users#show'
+
+  get 'users/:id/posts', to: 'posts#index'
+  get 'users/:id/posts/:id', to: 'posts#show'
```

Comment:
- [ ] [OPTIONAL] You have done a great job defining the index and show routes for the users and posts. However, there is a better way to handle this as shown below 👇 It is equivalent to what you already have, but with fewer lines of code and more possibilities 😎 
``` ruby
resources :users, only: [:index, :show] do
  resources :posts, only: [:index, :show]
end
```

Reply:
Thanks for the review and suggestion😁